lib/storage/src/content_manager/consensus/consensus_wal.rs

```diff
@@ -56,8 +56,51 @@ impl ConsensusOpWal {
         high: u64,
         max_size: Option<u64>,
     ) -> raft::Result<Vec<RaftEntry>> {
-        let mut entries = (low..high).map(|id| self.entry(id)).try_collect()?;
-        limit_size(&mut entries, max_size);
+        let mut size = 0;
+
+        let entries = (low..high)
+            .map(|id| self.entry(id))
+            .take_while(|entry| {
+                let entry = match entry {
+                    Ok(entry) => entry,
+                    Err(_) => return true,
```

Comment:
returning true means that errors are ignored right?

the previous implementation with `try_collect` would return the error in that I believe

Reply:
I believe that means, that "size" of errored entry is 0, so we don't need to check size. The error itself should be handled further